org.getmonero.Monero.yaml

```diff
@@ -0,0 +1,34 @@
+app-id: org.getmonero.monero.desktop
```

Comment:
Same comment as elsewhere. org.getmonero.Monero is the id, given the name the manifest.